Sources/SafeDICore/Models/Property.swift

```diff
@@ -61,6 +61,14 @@ public struct Property: Codable, Hashable, Comparable, Sendable {
         )
     }
 
+    var asTupleElement: TupleTypeElementSyntax {
+        TupleTypeElementSyntax(
+            firstName: .identifier(label),
+            colon: .colonToken(),
```

Comment:
I don't know why we don't need trailing trivia here but... we don't per the tests.